R/maraca.R

```diff
@@ -124,6 +144,35 @@ maraca <- function(
   )
 }
 
+##' @export
```

Comment:
CRAN will reject this because it has no documentation for the parameters or description. 
running R CMD check is useless. you must also add --as-cran and even then, they are still not happy.

Reply:
Added documentation